docs/nima/metrics/micrometer.adoc

```diff
@@ -0,0 +1,214 @@
+///////////////////////////////////////////////////////////////////////////////
+
+    Copyright (c) 2023 Oracle and/or its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+///////////////////////////////////////////////////////////////////////////////
+
+= Micrometer Metrics
```

Comment:
The Micrometer integration, at least as it is in 3.x, will probably go away. Our metrics implementation is likely to be quite different in 4.x. There will be (most likely) even better integration with Micrometer but accomplished in a much different way. So this page will either go away or be completely rewritten.

Reply:
I stripped the SE content and left it empty for 4. We can delete it if we don't need it.